paddle/fluid/framework/reader.h

```diff
@@ -25,8 +25,6 @@
 namespace paddle {
 namespace framework {
 
-enum ReaderStatus { kRunning, kStopped };
```

Comment:
kRunning and kStopped should not belong to namespace `framework`. Move this enum to ReaderBase